src/hooks/market/useMarkets.ts

```diff
@@ -1,22 +1,24 @@
 import { useContractRead, useContractReads } from "wagmi";
 import registryContractJson from "../../abi/Registry.json";
-import { ethers } from "ethers";
+import { BigNumber, ethers } from "ethers";
 import { useEffect, useState } from "react";
 import api from "../../apis/Api";
 
+const registryContractAddress =
+  process.env.REACT_APP_REGISTRY_CONTRACT ||
+  "0xd90AE997C32EdE8feCe39694460543868Da0d0D1";
```

Comment:
i think we should throw an error if a registry contract is now provided

Reply:
Let's not blow this PR out of scope of the issue much